main.c

```diff
@@ -53,6 +55,9 @@
 #include "libtcmu_config.h"
 #include "libtcmu_log.h"
 
+#define TCMUR_MIN_OPEN_FD 65536
+#define TCMUR_MAX_OPEN_FD 1048576
```

Comment:
@mikechristie @pkalever 
If this PR will be merged, since I wrote the configure code I can do this later based on it.